style.css

```diff
@@ -2,3 +2,218 @@
 /*font-family: 'Montserrat', sans-serif;
 font-family: 'Pacifico', cursive;
 font-family: 'Poppins', sans-serif;*/
+*,*:before, *:after {
+    padding: 0;
+    margin: 0;
+    border: 0;
+    box-sizing: border-box;
+}
+html, body {
+    height: 100%;
+    font-family: 'Poppins', sans-serif;
+}
+.container {
+    max-width: 1180px;
+    margin: 0px auto;
+    padding: 0px 10px;
+}
+.header {
+width: 100%;
```

Comment:
be careful to add the needed spacing to the left side here so that the file is easier to read

Reply:
ok, fixed it